src/index.md

```diff
@@ -12,6 +12,78 @@ The goal of this guide is to answer questions that developers of the JDK might h
 
 There are many common use cases that aren't detailed in the formal process. This guide suggests how to work in such cases.
 
+## I have a patch, what do I do?
+
+::: {.box}
+[Quick Links]{.boxheader}
+
+* [Oracle Contributor Agreement (OCA)](https://www.oracle.com/technical-resources/oracle-contributor-agreement.html)
+:::
+
+In many GitHub projects the standard way to propose a change is to create a pull request (PR) and discuss the patch in the PR. For OpenJDK projects the situation is somewhat different. First of all, the JDK and the surrounding tooling developed in the OpenJDK projects are part of commercial products maintained and sold by many companies. The JDK is installed on billions of devices and there are many millions of developers out there who use the Java language. For this reason there must be several safety checks before a change is made.
+
+### 1. Sign the OCA
+
+Oracle are the stewards of the OpenJDK project, and owns the Java brand. In order to make your patch available for review you must first sign the [Oracle Contributor Agreement](https://www.oracle.com/technical-resources/oracle-contributor-agreement.html) (OCA). This agreement gives Oracle and you as a contributor joint copyright interests in the code. You will retain your copyright while also granting those rights to Oracle. If you try to create a PR before you have signed the OCA, you will get instructions telling you to do so, and the PR will not be published until this is done.
```

Comment:
```suggestion
Oracle are the stewards of the OpenJDK project, and owns the Java brand. In order to make your patch available for review you must first sign the [Oracle Contributor Agreement](https://www.oracle.com/technical-resources/oracle-contributor-agreement.html) (OCA). This agreement gives Oracle and you as a contributor joint copyright interests in the code. You will retain your copyright while also granting those rights to Oracle. 

When you sign the OCA, please make sure that you specify your GitHub user name in the `Username` field of the OCA. If you try to create a PR before you have signed the OCA, or if you did not specify your GitHub user name, you will get instructions telling you to do so, and the PR will not be published until this is done. OCA registration is a manual process. Please allow for up to several days to have your OCA application processed, even though it's normally processed swiftly.
```

Reply:
Why do you propose to remove this sentence?